app/views/layouts/application.html.erb

```diff
@@ -88,6 +88,11 @@
     <% end %>
     <div id="message-notice"></div>
     <div id="page-content" class="universal-page-content-wrapper <%= view_class %>" data-current-page="<%= current_page %>">
+      <% if flash[:global_notice] %>
```

Comment:
It turned out that we didn't have visible flash messages on the main page, so those set up on deletion were useless. Including the one that had already existed (when destroying an account), so I had to implement visible flash messages.